paddlecloud/paddlejob/views.py

```diff
@@ -40,14 +41,21 @@ def post(self, request, format=None):
         dc = obj.get("datacenter")
         volumes = []
         for name, cfg in settings.DATACENTERS.items():
-            if cfg["type"] == "cephfs":
-                volumes.append(CephFSVolume(
-                    monitors_addr = cfg["monitors_addr"],
-                    user = cfg["user"],
-                    secret_name = cfg["secret"],
-                    mount_path = cfg["mount_path"] % username,
-                    cephfs_path = cfg["cephfs_path"] % username
-                ))
+            if dc == name:
+                if cfg["type"] == "cephfs":
+                    volumes.append(CephFSVolume(
```

Comment:
Check if `monitors_addr` `user` `secret ` exists. Return an error code when parameters not found.